server/app/repository/ApplicationRepository.java

```diff
@@ -58,18 +59,19 @@ public void forEachSubmittedApplication(Consumer<Application> fn) {
         .findEach(fn);
   }
 
-  /**
-   * Submit an application, which will delete any in-progress drafts, obsolete any submitted
-   * applications to a program with the same name (to include past versions of the same program),
-   * and create a new application in the active state.
-   */
+  @VisibleForTesting
```

Comment:
Is this needed given submitApplication is public?

Reply:
The annotation indicates why it is public and doesn't influence access level